app/javascript/modules/listingDetailsLottery/ListingDetailsLottery.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from "react"
+import dayjs from "dayjs"
+import { AppearanceSizeType, AppearanceStyleType, Button, t } from "@bloom-housing/ui-components"
+import { RailsListing } from "../listings/SharedHelpers"
+import { isLotteryComplete } from "../../util/listingUtil"
+import { getLotteryBucketDetails } from "../../api/listingApiService"
+import { RailsLotteryBucketsDetails } from "../../api/types/rails/listings/RailsLotteryBucketsDetails"
+import { ListingDetailsLotteryModal } from "./ListingDetailsLotteryModal"
+
+export interface ListingDetailsLotteryProps {
+  listing: RailsListing
+}
+
+export const ListingDetailsLottery = ({ listing }: ListingDetailsLotteryProps) => {
+  const [lotteryBucketDetails, setLotteryBucketDetails] = useState<RailsLotteryBucketsDetails>()
+  const [isModalOpen, setIsModalOpen] = useState(false)
+
+  useEffect(() => {
+    void getLotteryBucketDetails(listing.Id).then((lotteryBucketDetails) => {
+      setLotteryBucketDetails(lotteryBucketDetails)
+    })
+  }, [listing.Id])
+
+  return (
+    isLotteryComplete(listing) && (
+      <div className="border-b pt-4 text-center">
+        <h4 className="mb-2 text-2xl">{t("listings.lottery.lotteryResults")}</h4>
```

Comment:
We can likely using the `Heading` component here, so the styles stay consistent https://storybook.bloom.exygy.dev/?path=/story/headers-heading--base-4